meinberlin/apps/budgeting/templates/meinberlin_budgeting/proposal_list.html

```diff
@@ -55,7 +55,12 @@
                     </div>
                 </div>
             </div>
-            {% map_display_points object_list module.settings_instance.polygon %}
+            {% if module.blueprint_type == 'PB3' %}
+                {% has_or_would_have_perm 'meinberlin_budgeting.view_support' request.user module as may_view_support %}
+                {% map_display_points object_list module.settings_instance.polygon 'true' may_view_support|yesno:"false,true" %}
```

Comment:
This is a bit hacky with the [yesno template tag](https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#yesno), but did not really find a better solution for negating a bool in template :woman_shrugging: